raiden/network/transport/matrix/transport.py

```diff
@@ -395,6 +396,7 @@ def on_success(greenlet):
         self._client.sync_thread.link_exception(self.on_error)
         self._client.sync_thread.link_value(on_success)
         self.greenlets = [self._client.sync_thread]
+        self._schedule(30, self._address_mgr.get_address_mgr_info)
```

Comment:
Just a question for my understanding: why do we want to run this function twice, why only twice (compared to "every 30s), ond why after 30s?

Reply:
It's supposed to run every 30s, `_schedule` calls itself after 30s again

Reply:
Ah, my bad. Then I'm with Ulo in the bview that a while loop is easier to understand.